.travis.yml

```diff
@@ -14,7 +14,7 @@ notifications:
 
 env:
   - EMBER_TRY_SCENARIO=default
-  - EMBER_TRY_SCENARIO=ember-1.13
+  - EMBER_TRY_SCENARIO=ember-release
```

Comment:
On second thought, version accordingly and drop < 1.13 support so we can leverage the new attr hooks and remove the observers below in favor of that pattern.  It will be much cleaner and easier to maintain.